pkg/catalogv2/system/system.go

```diff
@@ -284,6 +286,17 @@ func (m *Manager) isInstalled(namespace, name, version string, values map[string
 			return false, err
 		}
 
+		if minVersion != "" {
+			min, err := semver.NewVersion(minVersion)
+			if err != nil {
+				return false, err
+			}
+			if min.LessThan(ver) || min.Equal(ver) {
+				logrus.Infof("Skipping installing/upgrading desired version %v for release %v, current version %v is greater or equal to minimal required version %v", desired.String(), name, ver.String(), minVersion)
```

Comment:
This will be annoying to see in the logs every 15 minutes.

Reply:
Will move this to debug.